player.py

```diff
@@ -192,12 +193,19 @@ def place_ant_on_anthill(self, ant_positions, anthill, ant):
       Newly updated version of the anthill list; showing one fewer None and one more 
       ant ID (string) in the appropriate place.
     """
-    for i in range(len(anthill)-1, -1, -1):
-      if anthill[i] is not None:
-        continue
-      else:
-        anthill[i] = ant
-        break
+    if anthill_order == "user choice":
+      i = int(
+        self.make_choice(
+          self.define_allowed_choices_anthill_placement(anthill), PROMPT_TEXT_ANTHILL_PLACEMENT_CHOICE
```

Comment:
When placing arguments in a different line to the function, I typically place each argument in its own line (final trailing comma is optional but I like it).
```suggestion
          self.define_allowed_choices_anthill_placement(anthill), 
          PROMPT_TEXT_ANTHILL_PLACEMENT_CHOICE,
```